### DISCUSSION TITLE
MacOS Scrolling issue

### DISCUSSION THREAD

**Discussion by Hexilitine** [ppy/osu#26791] [Q&A]
### Type

Crash to desktop

### Bug description

I first discovered it when I opened osu lazer for the first time, and when I scrolled to adjust to volume, the game just crashed as a whole. I re-installed it entirely just to make sure and it still stayed. I am on a MacOS Intel

This wasn't an issue in the old osu! client that on wineskin, and it's also ONLY scrolling that crashes straight into the desktop, auto report to apple everytime

### Version

v2024.114.0

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14090877/compressed-logs.zip)

**Comment by peppy** [comment]
I don't see any crash in logs so it's likely GPU related. Try changing your renderer to `OpenGL (Legacy)`.

**Reply by Hexilitine** [reply]
It already is, but it worked just fine on the old osu client using wineskin, which should be much heavier than this, and it's nearly always on 120fps even with native resolution
